src/ui/public/courier/fetch/request/segmented.js

```diff
@@ -61,12 +63,12 @@ export default function SegmentedReqProvider(es, Private, Promise, timefilter, c
       // parameters via the handle
       if (_.isFunction(this._initFn)) this._initFn(this._handle);
       return this._createQueue().then((queue) => {
+        if (this.stopped) return;
+
         this._all = queue.slice(0);
 
         // Send the initial fetch status
         this._reportStatus();
-
-        return super.start();
```

Comment:
Moving this may be a problem.  I'm not 100% on that, but I seem to remember there being a very specific reason why the parent constructor wasn't invoked until after the queue was created.  Gross as hell though, and I really should have left a comment about it in the code.